### ISSUE TITLE
Playground not starting

### ISSUE THREAD

**Issue by javobal** [boazpoolman/strapi-plugin-sitemap#128]
### Describe the bug

Not able to run the playground on a fresh fork of the repo

### Steps to reproduce the behavior

1. Fork the repo
2. execute yarn install on the root of the project and then yarn playground:install
3. execute yarn develop and then yarn playground:develop
4. See error for yarn playground:develop
5. Also tried doing yarn playground:start and i get also an error

### Expected behavior

I expect the playground to start just by following the readme instructions

### Screenshots
yarn develop to the left and yarn playground:develop to the right
<img width="1593" alt="image" src="https://github.com/boazpoolman/strapi-plugin-sitemap/assets/6923752/f546aa2b-ba15-478b-9fb8-4b0937c5fdd9">

### System

- Node.js version: 18.16.0
- NPM version: 9.5.1
- Strapi version: 4.11.4
- Plugin version: 3.0.0
- Database: N/A
- Operating system: MacOS

### Additional context

This is the first time i am trying to contribute to the project so sorry if this is something very obvious on my side

**Comment by javobal** [comment]
got it working by adjusting the plugin resolve path config. again i have very little experience with strapi but was i supposed to add a config folder and use it ? i noticed the resolve path probably works for these.

<img width="2268" alt="image" src="https://github.com/boazpoolman/strapi-plugin-sitemap/assets/6923752/f9e80564-1815-4652-b0da-d4a88d53cdd7">

thanks !

**Comment by javobal** [comment]
after that i was getting a sitemap generated but http://localhost:1337/api/sitemap/index.xml throwed an error until i copied the xsl folder into src/plugins/sitemap/xsl

this line 97 of server/controller/core.js can't find the xsl folder.

<img width="824" alt="image" src="https://github.com/boazpoolman/strapi-plugin-sitemap/assets/6923752/2cb4a0fa-fa53-45d4-82d3-604e03f55b01">

once i copied over the xsl folder i got the sitemap to appear in http://localhost:1337/api/sitemap/index.xml

**Comment by boazpoolman** [comment]
Thanks @javobal for testing this!
The playground is very new to this plugin. I added it just weeks ago.

I've tested around with the playground and experienced the same issues you did.
Based on your feedback I added these two commits which should fix them:

https://github.com/boazpoolman/strapi-plugin-sitemap/commit/c9c5db84e74f88a7244c147b32297f9ee5b68268
https://github.com/boazpoolman/strapi-plugin-sitemap/commit/81f51865fb4cbb3f8a30c22a4f3e52cf62f89338
